libbeat/autodiscover/providers/kubernetes/config.go

```diff
@@ -16,11 +16,6 @@ type Config struct {
 	SyncPeriod     time.Duration `config:"sync_period"`
 	CleanupTimeout time.Duration `config:"cleanup_timeout"`
 
-	IncludeLabels      []string `config:"include_labels"`
```

Comment:
Why remove config for labels and annotations?

Reply:
This change only moved them around in the code (while didn't change how they worked): https://github.com/elastic/beats/pull/7231/files#diff-f7068996b1041934b4fa71b630725b55R20